src/wallet.js

```diff
@@ -304,6 +321,16 @@ export class Wallet extends React.Component {
 
   componentDidMount() {
     this.props.store.onChange(this.onStoreChange);
+    const params = (new URL(document.location)).searchParams;
```

Comment:
Can you please create a section in the README that documents the expected search params.

Reply:
I'll change this to be exposed via window messages and will document the accepted message format